src/test/java/com/wikia/webdriver/elements/mercury/old/PortableInfoboxObject.java

```diff
@@ -337,22 +337,19 @@ public PortableInfoboxObject areDataValuesVisible() {
     return this;
   }
 
-  public PortableInfoboxObject areUnorderedListAndDataValuesMarginEqual() {
-    Assertion.assertEquals(
-        unorderedLists.get(0).getCssValue("margin"),
-        dataValues.get(0).getCssValue("margin")
-    );
-    PageObjectLogging.log("Unordered list labels and value", "have the same margin", true);
+  public PortableInfoboxObject verifyDataValueMargin() {
+    Assertion.assertEquals(dataValues.get(0).getCssValue("margin"), "0px");
+    PageObjectLogging.log("Data values", "have correct margin", true);
 
     return this;
   }
 
-  public PortableInfoboxObject areOrderedListAndDataValuesMarginEqual() {
-    Assertion.assertEquals(
-        orderedLists.get(0).getCssValue("margin"),
-        dataValues.get(0).getCssValue("margin")
-    );
-    PageObjectLogging.log("Ordered list labels and value", "have the same margin", true);
+  public PortableInfoboxObject verifyListMargin() {
+    Assertion.assertEquals(orderedLists.get(0).getCssValue("margin"), "0px 0px 8px 10px");
```

Comment:
One minor thing, maybe it is worth to extract these margin values to constants? It's just proposition, not pushing ;)